scripts/check_changelog_fragments.sh

```diff
@@ -56,22 +56,17 @@ while IFS= read -r fname; do
     exit 1
   fi
 
-  # if specified, this option validates that the contents of the news fragment
-  # contains a properly formatted authors line at the end of the file, generally
-  # used for external contributor PRs.
-  if [[ $1 == "--authors" ]]; then
-    last=$( tail -n 1 "${CHANGELOG_DIR}/${fname}" )
-    if [[ "${last}" == "authors: "*@* ]]; then
-      echo "invalid fragment contents: author should not be prefixed with @"
-      exit 1
-    elif [[ "${last}" == "authors: "*,* ]]; then
-      echo "invalid fragment contents: authors should be space delimited, not comma delimited."
-      exit 1
-    elif ! [[ "${last}" =~ ^(authors: .*)$ ]]; then
-      echo "invalid fragment contents: author option was specified but fragment ${fname} contains no authors."
-      exit 1
-    fi
-
+  # Each fragment should have a properly formatted authors line at the end of the file.
```

Comment:
Here you drop the check for `--authors` but the above places where this is called still specify `--authors`. Did you mean to drop the flag from the callers?

Reply:
Yes, I removed it https://github.com/vectordotdev/vector/pull/21463/commits/9f71c0b127dc128f040db009cee3eeed7fc16167.